docs/contributing.md

```diff
@@ -0,0 +1,88 @@
+All contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas are welcome.
+We recommend going through the [`issues`](https://github.com/pyomeca/pyomeca/issues) to find issues that interest you.
+Then, you can get your development environment setup with the following instructions.
+
+## Forking Pyomeca
+
+You will need your own fork to work on the code.
+Go to the [pyomeca project page](https://github.com/pyomeca/pyomeca/) and hit the `Fork` button.
+You will want to clone your fork to your machine:
+
+```bash
+git clone https://github.com/your-user-name/pyomeca.git
+cd pyomeca
+git remote add upstream https://github.com/pyomeca/pyomeca.git
+```
+
+## Creating a Python environment
+
+Before starting any development, you’ll need to create an isolated pyomeca development environment:
+
+- Install [miniconda](https://conda.io/miniconda.html)
+- `cd` to the pyomeca source directory
+- Install pyomeca dependencies with:
+
+```bash
+conda env create
+conda activate pyomeca
+pip install -r requirements-dev.txt
+```
+
+## Creating a branch
+
+You want your master branch to reflect only production-ready code, so create a feature branch for making your changes.
+For example:
+
+```bash
+git branch -b new-feature
+```
+
+Keep any changes in this branch specific to one bug or feature so it is clear what the branch brings to pyomeca.
+
+## Testing your code
+
+Adding tests is required if you add or modify existing codes in pyomeca.
+Therefore, it is worth getting in the habit of writing tests ahead of time so this is never an issue.
+The pyomeca test suite runs automatically on GitHub Actions, once your pull request is submitted.
+However, we strongly encourage running the tests prior to submitting the pull request.
+To do so, simply run `make test`.
```

Comment:
This one is problematic though, since here, from the PyCharm debugger, I could not run the tests...

Reply:
I added instruction to run tests without `make` in https://github.com/pyomeca/pyomeca/pull/97/commits/2e22b5a5518daaa9da06ef3ca6d3ccdd3de2d0ea.
However, in your case, it boils down to an editor configuration (see [here](https://www.jetbrains.com/help/pycharm/pytest.html)). Given the multiple editors, I'm not sure I want to give explanations for all of them. The user is free to use `pytest` — which is fairly standard — as he sees fit.